public/arrow_forward.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="48px" viewBox="0 0 24 24" width="48px" fill="#000000">
+<path d="M0 0h24v24H0V0z" fill="none"/>
+<path d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8-8-8z"/>
+</svg>
```

Comment:
Hmm, I'm hesitant about this one. I think rather than making our own icon svgs, we could use the plethoras of icon libraries such as [don't awesome](https://fontawesome.com/). What do you think @AaronCGoidel?

Reply:
Honestly, it comes down to how many icons we're using. I don't think its that many?

Reply:
I think there are quite a few. Some I can think of is arrows, menus, search icon, social media icons, etc.. It would be worth it to use a library rather than making an svg for every icon.

Reply:
For now we will keep using the svg, make to provide source for where you found it as a comment

Reply:
https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aarrow_forward%3A